test/e2e/ha/scale_core_contrail_services_test.go

```diff
@@ -317,6 +332,159 @@ func TestHACoreContrailServices(t *testing.T) {
 				}
 			})
 		})
+		t.Run("when manager resource is upgraded", func(t *testing.T) {
```

Comment:
Why is there an upgrade test in file `scale_core_contrail_services_test.go`? Isn't it tested in `upgrade_core_contrail_services_test.go`?

Reply:
"scale_core_contrail_services_test.go" tests for upgrade of all services at once. I had random results with it for Cassandra. "upgrade_core_contrail_services_test.go" tests for upgrade of one service at a time. This way the tests always failed for Cassandra.